store.js

```diff
@@ -1,24 +1,32 @@
 import { createStore, applyMiddleware, compose } from 'redux'
 import thunk from 'redux-thunk'
+import { createEpicMiddleware } from 'redux-observable'
 import { createLogger } from 'redux-logger'
 import { routerReducer, syncHistoryWithStore, routerActions, routerMiddleware } from 'react-router-redux'
 import { browserHistory } from 'react-router'
 
-import rootReducer from './reducer'
-import client from './client'
+import feathers from './client'
+import rootUpdater from './updater'
+import rootEpic from './epic'
 
 const middleware = [
-  thunk.withExtraArgument(client),
+  thunk.withExtraArgument(feathers),
```

Comment:
it's still used by authentication, yeah? [accounts/actions.js](https://github.com/dogstack/dogstack-example/blob/feathers-action/account/actions.js)